### DISCUSSION TITLE
Could we have the ability to establish repetition?

### DISCUSSION THREAD

**Discussion by eldaroid** [marcusolsson/obsidian-projects#954] [General]
Could we have the ability to establish repetition? 
every month every day

**Comment by provocateur-dev** [comment]
I think this would be a great addition. Just to clarify the conversation, are you talking repetition (one or more actions being repeated in a single project at a set interval) or recurrence (a new project starting on a regular interval)? I'd love to see both but I don't want to hijack your preference one way or the other.
